NEWS

```diff
@@ -1,3 +1,49 @@
+etcd v3.2.0 (TODO)
+- faster serializable reads
+- embedded etcd
+  - Etcd.Peers field is now []*peerListener
+  - add Election service
```

Comment:
election / lock service stuff seems like it would belong under a separate `RPCs` heading